tests/test_parser.py

```diff
@@ -52,7 +52,12 @@ def test_dockerfile_structure(self, dfparser):
                           " label  foo  \\\n",  # extra ws
                           "    # comment\n",    # should be ignored
                           "    bar  \n",        # extra ws, continuation line
-                          "USER  {0}".format(NON_ASCII)]   # extra ws, no newline
+                          "USER  {0}\n".format(NON_ASCII),
+                          "# comment \\\n",    # extra ws
```

Comment:
How about adding a test with extra whitespace *after* the backslash as well? (You'll be surprised what docker does with that...)